### PR TITLE
Add from and to query to orders api and add image_url to product

### PR DESCRIPTION
**Pull request by JonaMata**

### CODE REVIEW

File: app/Http/Controllers/UserApiController.php [hunk 1]
```diff
@@ -62,9 +63,21 @@ public function getAchievements()
     /**
      * @return Collection|OrderLine[]
      */
-    public function getPurchases()
+    public function getPurchases(Request $request)
```
Comment (Jessseee) [inline app/Http/Controllers/UserApiController.php:66]: Does this not break any other uses of this method?

**Review by Jessseee — CHANGES_REQUESTED** [review]

File: app/Http/Controllers/UserApiController.php (hunk 1 above)
Comment (JonaMata) [reply app/Http/Controllers/UserApiController.php:66]: Are there other usages?

Comment (Jessseee) [reply app/Http/Controllers/UserApiController.php:66]: Yeah, not sure, just checking.

**Review by Jessseee — APPROVED** [review]
LGTM

**Pull request merged**